subhub/customer.py

```diff
@@ -80,16 +80,21 @@ def existing_or_new_customer(
         return create_customer(
             subhub_accouunt, user_id, email, source_token, origin_system, display_name
         )
+    logger.info("existing customer", account=db_account)
     customer_id = db_account.cust_id
     return existing_payment_source(customer_id, source_token)
 
 
 @timed
 def existing_payment_source(customer_id: str, source_token: str) -> stripe.Customer:
     existing_customer = stripe.Customer.retrieve(customer_id)
-    if not existing_customer["sources"]["data"]:
-        existing_customer = stripe.Customer.modify(customer_id, source=source_token)
-        logger.info(f"add source {existing_customer}")
+    logger.info("existing customer", data=existing_customer)
```

Comment:
`existing_customer=existing_customer`?

Reply:
valid, fixed.

Reply:
is this fixed? seems to still have `data=existing_customer`; the pattern im advocating here is to keep it consistent with the rest of the code base where we generally use the same name; notable exceptions being `error=e` and the like.